### PR TITLE
Cluster form

### PR DESCRIPTION
**Pull request by guillaumev**

### CODE REVIEW

File: src/components/HidContacts.js [hunk 1]
```diff
@@ -72,10 +86,9 @@ class HidContacts extends React.Component {
       <MaterialAsyncSelect
         isMulti={this.props.isMulti}
         loadOptions={this.getOptions}
-        getOptionLabel={(option) => {return option.name}}
```
Comment (guillaumev) [inline src/components/HidContacts.js:75]: @barbaraReliefapps Why did you need these changes on HidContacts ?

File: src/components/MaterialAsyncSelect.js [hunk 2]
```diff
@@ -75,8 +75,8 @@ function SelectWrapped(props) {
     styles={customStyles}
     isClearable={true}
     isMulti={props.isMulti}
-    getOptionValue={(option) => { if (props.getOptionValue) { return props.getOptionValue(option) } else { return option.id } }}
-    getOptionLabel={(option) => { if (props.getOptionLabel) {return props.getOptionLabel(option)} else {return option.label}}}
+    getOptionValue={(option) => option.id}
```
Comment (guillaumev) [inline src/components/MaterialAsyncSelect.js]: @barbaraReliefapps Same question here: why did you make these changes on MaterialAsyncSelect ?

File: src/components/HidContacts.js (hunk 1 above)
Comment (bamoure) [reply src/components/HidContacts.js:75]: I was testing with the Contacts and since the HIDContact field was later deleted I forgot to remove my changes, modifying it in new commit

File: src/components/MaterialAsyncSelect.js [hunk 3]
```diff
@@ -159,8 +159,6 @@ class MaterialAsyncSelect extends React.Component {
 	              simpleValue: true,
 	              loadOptions: this.props.loadOptions,
 				        value: this.props.value,
-                getOptionLabel: this.props.getOptionLabel,
```
Comment (guillaumev) [inline src/components/MaterialAsyncSelect.js]: @barbaraReliefapps I think this needs to be readded as well...

File: src/containers/EventForm.js [hunk 4]
```diff
@@ -168,14 +165,14 @@ class EventForm extends React.Component {
...
-          {/* Address */}
-              <FormControl fullWidth margin = "normal">
-                <FormLabel>Venue</FormLabel>
-                <Address onChange={(s) => this.props.handleSelectChange('address', s)} value={this.props.doc.address} />
-                <FormHelperText id = "address-text">
-                  Indicate here where the {this.props.label} takes place.
-                </FormHelperText>
-              </FormControl>
+        {/* Address */}
+            <FormControl required fullWidth margin = "normal">
```
Comment (guillaumev) [inline src/containers/EventForm.js]: @barbaraReliefapps The address should not be required
